run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -ux
+status=0
+
+# global exports
+export PGPORT=55435
+
+# build extension
+make install
+
+# initialize database
+initdb -D $PGDATA
+
+# change PG's config
+echo "port = $PGPORT" >> $PGDATA/postgresql.conf
```

Comment:
No need for that, you can pass  -o "-p $PGPORT"  parameter to pg_ctl

Reply:
thanks, didn't know that. Will fix